llvm/include/llvm/ADT/GenericUniformityImpl.h

```diff
@@ -33,6 +33,12 @@
 /// the propagation of the impact of divergent control flow on the divergence of
 /// values (sync dependencies).
 ///
+/// NOTE: In general, no interface exists for a transform to update
+/// (Machine)UniformityInfo. Additionally, (Machine)CycleAnalysis is a
+/// transitive dependence, but it also does not provide an interface for
+/// updation. Given that, transforms should not preserve uniformity in their
```

Comment:
"updation" should be updating?

Reply:
"updation" should be a word! :-)

Reply:
I had looked it up ... "updation" is a word, and it's use as a noun is not wrong here. "updating" requires an object after it. But it was not clear to me if the verb "updating" is preferred over the noun "updation" in general ... so went with @arsenm's preference, writing it off as an ESL thing, and/or a tussle between en_US, en_UK and en_IN :)